client/state/action-types.js

```diff
@@ -67,7 +67,9 @@ export const CURRENT_USER_FLAGS_RECEIVE = 'CURRENT_USER_FLAGS_RECEIVE';
 export const CURRENT_USER_ID_SET = 'CURRENT_USER_ID_SET';
 export const DESERIALIZE = 'DESERIALIZE';
 export const DIRECTLY_ASK_QUESTION = 'DIRECTLY_ASK_QUESTION';
-export const DIRECTLY_INITIALIZE = 'DIRECTLY_INITIALIZE';
+export const DIRECTLY_INITIALIZATION_START = 'DIRECTLY_INITIALIZATION_START';
+export const DIRECTLY_INITIALIZATION_SUCCESS = 'DIRECTLY_INITIALIZATION_SUCCESS';
+export const DIRECTLY_INITIALIZATION_ERROR = 'DIRECTLY_INITIALIZATION_ERROR';
```

Comment:
Ha — my brain is thinking "state machine" so naturally `START` should come first right??? :P